.jenkins/pytorch/win-build.sh

```diff
@@ -56,6 +56,8 @@ if [[ $PYLONG_API_CHECK == 0 ]]; then
 fi
 set -ex
 
+echo 'python %*' > /c/Windows/py.bat
+
```

Comment:
Can you please explain, why this change is needed?

Reply:
So flatbuffers CMakeList.txt calls the 'py' command which is not available. Discussed internally: https://fb.workplace.com/groups/1274980366011614/permalink/2066627280180248/

Reply:
Can you please share a reference to the source code in flatbuffers that calls that?

Reply:
https://github.com/google/flatbuffers/blob/master/CMakeLists.txt#L533